core/src/main/scala/org/apache/spark/sql/hive/SnappyStoreHiveCatalog.scala

```diff
@@ -626,6 +671,32 @@ class SnappyStoreHiveCatalog(context: SnappyContext)
     }
   }
 
+  class SnappyIndexHandler extends TableBasedIndexHandler {
+    override def usesIndexTable() : Boolean = false
+
+    @throws(classOf[HiveException])
+    override def analyzeIndexDefinition(table: Table, index: Index, table1: Table) : Unit = { }
+
+    def generateIndexQuery(var1: List[Index], var2: ExprNodeDesc,
+                           var3: ParseContext, var4: HiveIndexQueryContext): Unit = {}
+
```

Comment:
Is this class required? I thought you mentioned in the PR description that it is not.